donut/modules/bodfeedback/helpers.py

```diff
@@ -0,0 +1,255 @@
+from donut import email_utils
+from donut.modules.bodfeedback import email_templates
+import flask
+import pymysql.cursors
+
+
+def send_update_email(email, complaint_id):
+    '''
+    Sends an email to [email] which should be an
+    email address of a user as a string
+    '''
+    msg = email_templates.added_message.format(get_link(complaint_id))
+    subject = "Received BoD Feedback"
+    email_utils.send_email(email, msg, subject)
+
+
+def register_complaint(data, notification=True):
```

Comment:
do you send an email to bod to tell them a complaint has been registered? what are reasons for or against this? what is the behavior of bodfeedback form on the legacy website? is the desired behavior for bod for them to check the listings during their weekly meetings? 🥇